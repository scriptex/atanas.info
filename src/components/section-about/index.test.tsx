import * as React from 'react';
import { shallow } from 'enzyme';

import { SectionAbout, LastAbout } from '.';

describe('SectionAbout component', () => {
	it('Should render the SectionAbout component', () => {
		const wrapper = shallow(<SectionAbout />);

		expect(wrapper).toMatchSnapshot();
	});
});

describe('LastAbout component', () => {
	it('Should render the LastAbout component', () => {
		const wrapper = shallow(<LastAbout />);

		expect(wrapper).toMatchSnapshot();
	});
});
