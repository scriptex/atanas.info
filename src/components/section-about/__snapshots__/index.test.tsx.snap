// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionAbout component Should render the SectionAbout component 1`] = `
<Component
  className=" c-article fullsize-background"
  hasButton={true}
  hasShell={true}
  id="about"
  style={
    Object {
      "backgroundImage": "url(images/temp/desktop.jpg)",
    }
  }
>
  <h2>
    About me
  </h2>
  <img
    alt="Atanas Atanasov smiling dressed in a green t-shirt"
    height="240"
    src="images/temp/atanas.jpg"
    width="240"
  />
  <div
    className="c-article__block"
    key="0"
  >
    <h3>
      2010
    </h3>
    <p
      key="0"
    >
      My name is Atanas and I am a senior JavaScript/TypeScript developer.
    </p>
    <p
      key="1"
    >
      I am currently located in Varna, Bulgaria. That's on the south-east side of Europe, on the Black Sea.
    </p>
    <p
      key="2"
    >
      I started my web development career in 2010 as a freelance apprentice to a hardcore fullstack web developer and pretty fast learned the basics of HTML and CSS.
    </p>
    <p
      key="3"
    >
      We had our own small web development company. In "HTML Lab" I was doing the front-end part and was mainly focused on slicing design files and turning them into static HTML pages.
    </p>
  </div>
  <div
    className="c-article__block"
    key="1"
  >
    <h3>
      2012
    </h3>
    <p
      key="0"
    >
      Between 2012 and 2018 I was employed in one of the larges web development companies in Varna, Bulgaria - 2create. In this company I worked for some of the biggest PSD-to-HTML brands out there - htmlBurger and htmlBoutique. I developed thousands (yes thousands) of different projects for clients from all over the globe and was focused on the front-end part of all projects.
    </p>
    <p
      key="1"
    >
      In the last 3 years of my employement there I was a direct asistant to several project managers and I also mentored several junior and intermediate front-end developers.
    </p>
    <p
      key="2"
    >
      During the period I participated in defining, integrating and teaching the internal coding standards in the company and co-authored the internal front-end projects starting template which was based on Webpack, Babel, PostCSS, SCSS and many more. The coding standards and Jarvis - that was the name of the front-end template - were (and are) used in tens of thousands of projects for clients from around the world.
    </p>
  </div>
  <div
    className="c-article__block"
    key="2"
  >
    <h3>
      2014
    </h3>
    <p
      key="0"
    >
      During my employment between 2012 and 2018 I developed, integrated and maintained the internal build systems for the company. This included mastering CSS pre- and post-processors such as LESS, SASS/SCSS and PostCSS, task runners such as Gulp and Grunt, module bundlers such as Webpack and Rollup, and NodeJS which I used as a runtime.
    </p>
    <p
      key="1"
    >
      Along these technologies I mastered JavaScript and it became my preferred coding language. I also presented a keynote on the features introduced in 2015 with the 6th version of ECMAScript (the JavaScript standard).
    </p>
    <p
      key="2"
    >
      In 2017 I started learning TypeScript which later became the number one tool in my utilities belt. I prefer starting new projects entirely in TypeScript regardless of the targeted platform or end - Web/Native or front-end/back-end.
    </p>
  </div>
  <div
    className="c-article__block"
    key="3"
  >
    <h3>
      2018
    </h3>
    <p
      key="0"
    >
      Since the beginning of 2018 I am working as a JavaScript/TypeScript engineer in a small startup located in Varna and operating internationally.
    </p>
    <p
      key="1"
    >
      At Three 11 we are focused on providing the best possible quality while using bleeding edge technologies. We are working on several in-house projects along with some interesting client websites and applications.
    </p>
    <p
      key="2"
    >
      We prefer JavaScript and TypeScript and utilize their power when building applications using React, Angular, Vue, React Native and even static web sites or WordPress powered CMS websites and blogs.
    </p>
  </div>
  <div
    className="c-article__block"
    key="4"
  >
    <h3>
      Today
    </h3>
    <p
      key="0"
    >
      Now I am a JavaScript and TypeScript developer with more than 10 years of professional experience. I am focused on the front-end but using a lot of back-end technologies as well. My daily tasks include coding HTML, CSS, JavaScript, TypeScript, PHP, Bash and many more.
    </p>
    <p
      key="1"
    >
      The latest project I worked on is a large international IoT application for one of the world's largest investor-owned electric utility service providers which aims to provide users from all over Europe with the ability to control their homes' smart appliances and energy consumption.
    </p>
  </div>
  <div
    className="c-article__block"
  >
    <h3>
      Current occupation
    </h3>
    <ul>
      <li>
        A Lead front-end developer at E.ON working on the
         
        <strong>
          <Component
            href="https://www.red-dot.org/project/eon-home-49256"
          >
            award winning
          </Component>
        </strong>
         
        web and mobile smart energy managemet application E.ON Home.
      </li>
      <li>
        A Senior software developer at
         
        <strong>
          <Component
            href="https://www.toptal.com/resume/atanas-atanasov"
          >
            Toptal
          </Component>
        </strong>
      </li>
      <li>
        A Founding member and Senior JavaScript/TypeScript enginner at
         
        <strong>
          <Component
            href="https://app.usebraintrust.com/talent/782/"
          >
            Braintrust
          </Component>
        </strong>
      </li>
    </ul>
    <br />
    <br />
    <h3>
      Connect with me
    </h3>
    <p>
      I am open for hire to work on your awesome idea. If you think I can help you, do not hesitate and
       
      <strong>
        <Component
          href="mailto:hi@atanas.info"
        >
          drop me a line
        </Component>
      </strong>
      .
    </p>
    <p>
      I also like to contribute to open-source projects and I have created some myself. Feel free to contact me if you want to talk about your open-source projects or
       
      <strong>
        <Component
          href="https://github.com/scriptex"
        >
          mine
        </Component>
      </strong>
      .
    </p>
  </div>
</Component>
`;
